telemetry/definitions/commonDefinitions.json

```
@@ -1123,7 +1125,7 @@
         },
         {
             "name": "cloudwatchlogs_filter",
-            "description": "Filters a CloudWatch Logs entity.",
+            "description": "Filters a CloudWatch Logs entity. Deprecated: use cloudwatchlogs_open instead.",
```

I believe `cloudwatchlogs_filter` is emitted in VS when users filter logs already opened in the viewer. So this event is still relevant because that information isn't captured in `cloudwatchlogs_open`. 

```suggestion
            "description": "Emitted when applying a filter to an already opened CloudWatch Logs entity",
```

---

+1, VS provides the ability to filter logs once the stream/group has been opened for viewing in the IDE.

---

> `cloudwatchlogs_filter` is emitted in VS when users filter logs already opened in the viewer

We also have that feature in vscode, but https://github.com/aws/aws-toolkit-vscode/pull/3410 changes the logic to capture that as `cloudwatchlogs_open` with `source=Editor` + these new fields.

---

> with source=Editor + these new fields.

Oh I didn't notice that we're changing `source` to reflect this. That works too although it may not be as obvious.

> The API calls are essentially the same, no? The only difference is the "source" of the action.

I wouldn't focus too much on the API calls. The events should reflect user actions/experiences. I'd rather have duplication than ambiguous data.